group_vars/galaxyservers.yml

```diff
@@ -225,9 +225,9 @@ galaxy_config:
     # len_file_path: /mnt/galaxy-app/tool-data/len
     #
     
-    # enable_oidc: true
-    # oidc_config_file: config/oidc_config.xml
-    # oidc_backends_config_file: config/oidc_backends_config.xml
+    enable_oidc: true
+    oidc_config_file: config/oidc_config.xml
```

Comment:
`oidc_config_file: "{{ galaxy_config_dir}}/oidc_config.xml"`
`oidc_backends_config_file: "{{ galaxy_config_dir}}/oidc_backends_config.xml"`

Reply:
Thanks for catching that, will fix.